.travis.yml

```diff
@@ -4,16 +4,32 @@ node_js:
 
 sudo: false
 
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - libcairo2-dev
+      - libjpeg8-dev
+      - libpango1.0-dev
+      - libgif-dev
+      - g++-4.9
+env:
+  - CXX=g++-4.9
+
+before_install:
+  - npm explore npm -g -- npm install node-gyp@latest
+
 branches:
   only:
   - master
 
 script:
   - npm test
   - npm run demo:build
-deploy:
-  provider: pages
-  skip-cleanup: true
-  local-dir: demo
-  on:
-    branch: master
+# deploy:
```

Comment:
Curious what the reason was for commenting out the gh-pages deploy...?

Reply:
It doesnt work yet. pending being able to actually set it up properly within the travis-github app.